tests/testHsm.py

```diff
@@ -134,6 +134,13 @@
     [58.5008586442, 28.2850942049],
 ])
 
+round_moments_expected = np.array([# sigma, e1, e2, flux, x, y
+    [2.40270376205, 0.197810277343, -0.372329413891, 3740.22436523, 36.4032272633, 20.4847916447],
+    [1.89714717865, 0.046496052295, -0.0987404286861, 776.709594727, 20.2893584046, 25.4230368047],
+    [1.77995181084, 0.0416346564889, -0.143147706985, 534.59197998, 9.51994111869, 12.6250775205],
+    [1.46549296379, -0.0831127092242, -0.0628845766187, 348.294403076, 20.6242279632, 39.5941625731],
+    [1.64031589031, 0.0867517963052, 0.0940798297524, 793.374450684, 58.4728765002, 28.2686937854],
+])
```

Comment:
I think flake8 is going to want two blank lines between this and the following `def`.